functions/shared_libs/api_clients/zoom_client.py

```
@@ -56,78 +91,499 @@ def fetch_access_token(self):
+    # ------------------------------------------------------------------
+    # Low-level HTTP wrappers
+    # ------------------------------------------------------------------
+
+    def _auth_headers(self) -> Dict[str, str]:
+        return {"Authorization": f"Bearer {self.access_token}"}
+
+    def _get(
+        self, url: str, params: Optional[Dict[str, Any]] = None
+    ) -> requests.Response:
+        self.validate_token()
+        return requests.get(url, headers=self._auth_headers(), params=params)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)